scripts/cc-cli/src/test/unit-tests/collect-coins.test.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  isExternalAddressValid,
+  isTxHashValid,
+} from "../../commands/collectCoins";
+
+describe("Test isTxHashValid", () => {
```

Comment:
Naming can be improved. (let me know if you want me to follow up with changes here). 

- remove "Test " - no need for it because we know we're already doing testing. The name of the function under test should be enough.  I also think that you can pass `isTxHashValid` directly as a name here, not as a string but IDK what the results will be. 
-